package vnpt.trust_call.handler.exceptions

internal class SdkIncompatibilityException : Exception()